x/consensus/keeper/keeper.go

```diff
@@ -65,7 +64,7 @@ var _ types.MsgServer = Keeper{}
 
 func (k Keeper) UpdateParams(ctx context.Context, msg *types.MsgUpdateParams) (*types.MsgUpdateParamsResponse, error) {
 	if k.GetAuthority() != msg.Authority {
-		return nil, errors.Wrapf(govtypes.ErrInvalidSigner, "invalid authority; expected %s, got %s", k.GetAuthority(), msg.Authority)
+		return nil, fmt.Errorf("invalid authority; expected %s, got %s", k.GetAuthority(), msg.Authority)
```

Comment:
I think @kocubinski as well adds the original error msg as text: https://github.com/cosmos/cosmos-sdk/blob/3db9528efb5fec1cccdb4e6f084c24ed195951b1/x/auth/keeper/msg_server.go#L27

Reply:
However, I actually think it makes more sense without it, as authority could be any account, and not just gov, so it is confusing.

Reply:
my thoughts while reading this as well